1.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="1.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>WEBSITES</title>
+</head>
+<body>
+    <header>
+        <div class="large-screen">
+           <P>PSD WEBSITES</P>
+            <P>UI KITS</P>
+            <P>PREMIUM PSD FILES</P>
+            <P>ABOUT</P>
+            <P>BLOG</P>
+         </div>
+         <div class="small-screen">
+            <div><p>
+                 Navigation
+             </p>
+             </div> 
+             <div class=MenuIcon>
+                <div></div>
+                <div></div>
+                <div></div>
+             </div>
+         </div>
+ 
+    </header>
+        <div class="banner">
+            <p>Awesome Free PSD<br>
```

Comment:
Semantically, it's not okay. 'Awesome Free PSD' seems like a title, so it's better to use h1 h2 h3